packages/zendesk-client/src/util.tsx

```diff
@@ -1,10 +1,25 @@
 import config from '@automattic/calypso-config';
 import { isInSupportSession } from '@automattic/data-stores';
 
+const IS_TEST_MODE_ENVIRONMENT = true;
+const IS_PRODUCTION_ENVIRONMENT = false;
+const PRODUCTION_ENVIRONMENTS = [ 'production', 'wpcalypso' ];
```

Comment:
We should probably include 'desktop' here I believe

Reply:
Not 100% sure, but maybe there is no env file.

We do an override [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-config/src/index.ts#L44), taking values from `window.configData` defined [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-config/src/desktop.ts). (You can see `env_id: 'desktop'` there)

Reply:
I see, thanks for pointing it out!